wiremock/mappings/HmppsAuthAndDelius_GetUser_TEMPORARY-ACCOMMODATION-E2E-TESTER.json

```diff
@@ -0,0 +1,23 @@
+{
+  "priority": 2,
+  "request": {
+    "method": "GET",
+    "urlPattern": "(?i)\/user\/TEMPORARY-ACCOMMODATION-E2E-TESTER"
+  },
+  "response": {
+    "transformers": ["response-template"],
+    "status": 200,
+    "jsonBody": {
+      "userId": 2500077027,
+      "username": "TEMPORARY-ACCOMMODATION-E2E-TESTER",
+      "firstName": "temporary-accommodation-e2e-tester",
+      "surname": "accommodation-e2e-tester",
+      "email": "accomm-e2e-testerw@example.digital.justice.gov.uk",
+      "enabled": true,
+      "roles": ["ROLE_PROBATION"]
```

Comment:
Same as above

Reply:
I'll use `CAS3_ASSESSOR ` as I believe this guys is the assessor user

Reply:
have stuck with `ROLE_PROBATION` here also - see above from @daveawc